### ISSUE TITLE
EndEvent is not thrown

### ISSUE THREAD

**Issue by Vyperus** [process-engine/process_engine#91]
#### Describe your issue

EndEvent is not thrown! See https://github.com/process-engine/bpmn-studio/issues/318

I debugged a little bit further into this. Event though i can't go "deeper" than the messageBus in BPMN-Studio, but it seems like the event simply isn't thrown.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by ElRaptorus** [comment]
@5ebastianMeier 
I can now say with some certainty that this is an issue within the process engine package itself. I have to do some further debugging, but I suspect a subscription to the messagebus for the process instance is broken.

**Comment by ElRaptorus** [comment]
@Vyperus @5ebastianMeier 
Found it. 
Somebody added a convenience method to the process engine service, for starting a process instance through the bpmn-studio-client.
This method is simply called "start" and addresses the `ProcessDefEntityTypeService` **directly** to start a process instance.
The problem with this approach is, quite simply, that it completely bypasses all logic required to properly start a process - like creating messagebus subscriptions.

This means that messages created during process execution will never be going anywhere, because no channel exists for them on the messagebus service.

We usually use the methods `executeProcess` and `executeProcessInstance` for starting a process, because it handles all the messagebus-related operations. Might be a good idea to refactor the bpm-studio client to do the same.

/cc @cmg-dev

**Comment by ElRaptorus** [comment]
**NOTE**:
I fixed this issue in the backend (it wasn't actually a fix, I just removed a pointless convenience method that circumvented the desired default behavior for starting processes).

But this problem also occured, in part, because of the usage of the obsolete bpmn-studio client. Since this client is going to be killed off anyway, I won't be doing any fixes in there.
@cmg-dev @Vyperus @5ebastianMeier
